ukvm/ukvm-configure

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+if [ "$#" -lt 1 ]; then
+    echo "Usage: ukvm-configure UKVM_SRC [MODULES]"
+    echo "    UKVM_SRC is /path/to/ukvm"
+    echo "    MODULES can be any combination of: net blk gdb"
+    exit
+fi
+
+UKVM_SRC=`readlink -e $1`
+shift
+UKVM_MODULES=$@
+
+(
```

Comment:
Yes, the suggestion is to have one Makefile.ukvm in all cases.

Ah, if you decide to use the example, you will have to change the names to UKVM_OBJECTS (and make the MODULES upper-case). Thanks @mato